app/user/groups/group-table.tsx

```
@@ -0,0 +1,324 @@
+        onClose={() => setIsEditModalOpen(false)}
+        onSave={handleEditGroup}
+      />
+    </>
+  )
+}
+
+export function GroupTable() {
+  const [groups, setGroups] = useState<Group[]>([])
+  const [loading, setLoading] = useState(true)
+  const [searchQuery, setSearchQuery] = useState("")
+  const [isNewGroupDialogOpen, setIsNewGroupDialogOpen] = useState(false)
```

ESLint

[error] 'isNewGroupDialogOpen' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)